Sources/Easy-Purchase-Receipt-Validator/ASN1/ASN1Decoder.swift

```
@@ -252,17 +257,19 @@ extension ASN1Decoder {
      
      So, in the code `let octetsToRead = first! - 0x80`, the subtraction of `0x80` is a way to extract the count of additional bytes used in the `long` form to represent the `length`. This value (`octetsToRead`) is then used to read the actual bytes representing the `length` from the iterator.
      */
-    func getContentLength(iterator: inout Data.Iterator) throws -> UInt64 {
+    func getContentLength(iterator: inout Data.Iterator, totalASN1ObjInUInt8Form: inout [UInt8]) throws -> UInt64 {
         let firstByte = iterator.next()
         guard let firstByte = firstByte else {
             return 0
```

As the function throws, can you please check if there is any benefit to throw error instead of returning 0.
for example : Can we return ParsingError.firstByteNotAvailable.

---

It should be 0. So that the content length will be 0 here.